index.js

```diff
@@ -204,12 +205,18 @@ function newConsole(self){
 }
 
 
+// Get info from a connected console
+function getConsoleInfo(self) {
```

Comment:
I disagree with this floating function, especially since its only called the one time.  Its internal should just be called directly in the socket.on connect.

Reply:
This function will be expanded heavily to get many more details about the console in the future.